pkgs/applications/editors/vscode-oss/default.nix

```diff
@@ -0,0 +1,214 @@
+{ stdenv, fetchFromGitHub, python, zip, makeWrapper, fetchurl, pkgconfig, jq
+, makeDesktopItem, runtimeShell, writeScript
+, nodejs, yarn, libsecret, xorg, ripgrep, electron
+, productOverride ? {}
+}:
+
+let
+  shortName = productOverride.nameShort or "Code - OSS";
+  longName = productOverride.nameLong or "Code - OSS";
+  executableName = productOverride.applicationName or "code-oss";
+
+  vscodePlatforms = {
+    i686-linux = "linux-ia32";
+    x86_64-linux = "linux-x64";
+    armv7l-linux = "linux-armv7l";
+    aarch64-linux = "linux-arm64";
+    x86_64-darwin = "darwin-x64";
+  };
+
+  system = stdenv.hostPlatform.system;
+
+  platform = vscodePlatforms.${system} or (throw "Unsupported platform: ${system}");
+
+in stdenv.mkDerivation rec {
+  pname = "vscode";
```

Comment:
Shouldn't this be `vscode-oss`? Well, unless we rename `vscode` to `vscode-bin` and `vscode-oss` to `vscode`.

Reply:
agreed